src/components/shared/button-library.module.css

```diff
@@ -38,9 +38,17 @@
 }
 
 .icon {
+    width: 12px !important;
+    min-width: 12px !important;
+    height: 12px !important;
```

Comment:
i'm ok with these importants because i'm creating a library here that overrides the defaults from antd, but is consistent through the design.